src/main/scala/ztis/model/RandomPrediction.scala

```diff
@@ -0,0 +1,33 @@
+package ztis.model
+
+import org.apache.spark.SparkContext._
+import org.apache.spark.mllib.recommendation.Rating
+import org.apache.spark.rdd.RDD
+
+import scala.util.Random
+
+class RandomPrediction(trainingData: RDD[Rating]) extends Predictor with Serializable {
```

Comment:
Ok, this is probably because you are referencing trainingData passed as constructor parameter - I would refactor it so you are accepting already constructed Array[(Double, Int)] because you are already doing this and not using the param in any other way